README.md

```diff
@@ -2,6 +2,39 @@
 
 Grant the AI octopus access to a portion of your desktop with a Model Context Protocol (MCP) server for the GNOME desktop.
 
+## Building
+
+To build the project, simply use:
+
+```bash
+cargo build
+```
+
+This will produce a debug binary under `target/debug/gnome-mcp-server`.
```

Comment:
Maybe we can recommend install it directly?

Reply:
I just pushed some changes that do exactly that. Have a look when you have a chance.